### ISSUE TITLE
ES2015 Export named `module` exported with the wrong name

### ISSUE THREAD

**Issue by natefaubion** [babel/babel#1544]
Given the following:
```js
export function module() {

}
```

When compiling to CJS, Babel renames to:
```js
exports._module = _module;
function _module() {

}
```

But when importing:
```js
import { module } from 'foo'
module()
```

Babel maps it to:
```js
_foo.module()
```

Which results in an `undefined is not a function` error.

**Comment by sebmck** [comment]
This would be due to #645.

**Comment by sebmck** [comment]
Fixed as of 5.4.3, thanks!
